modules/yieldmoBidAdapter.js

```diff
@@ -312,6 +313,16 @@ function openRtbImpression(bidRequest) {
   return imp;
 }
 
+function getBidFloor(bidRequest, mediaType) {
+  let floorInfo = {};
+
+  if (typeof bidRequest.getFloor === 'function') {
+    floorInfo = bidRequest.getFloor({ currency: CURRENCY, mediaType, size: '*' });
+  }
+
+  return floorInfo.floor || bidRequest.params.bidfloor || bidRequest.params.bidFloor || 0;
```

Comment:
you duplicated your check on bidRequest.params.bidFloor. Should not cause an error, however it's unnecessary.

Reply:
It's `bidfloor` vs `bidFloor`, they are a bit different

Reply:
Got it. I didn't catch the case.